src/screens/LinkContextScreen/elements/ContentRelated.js

```diff
@@ -1,66 +1,57 @@
-import { colors } from '../../../utils/colors';
-import { fonts } from '../../../utils/fonts';
+import {Gap} from '../../../components';
+import {COLORS, SIZES} from '../../../utils/theme';
+import {fonts} from '../../../utils/fonts';
 
-const ContentRelated = ({ item, onContentPressed }) => (
-    <Pressable onPress={() => onContentPressed(item)}>
-        <Text style={styles.domainItemTitle}>{item.content.title}</Text>
-        {item.content.image ? (
-            <Image
-                source={{ uri: item.content.image }}
-                style={styles.domainImage}
-            />
-        ) : (
-            <Image
-                source={NewsEmptyState}
-                style={styles.domainImageEmptyState}
-            />
-        )}
-        <View style={styles.domainTextContainer}>
-            <Text style={styles.domainItemDescription}>
-                {item.content.description}
-            </Text>
-        </View>
-        {/* <Gap height={14} /> */}
-    </Pressable>
+const ContentRelated = ({item, onContentPressed}) => (
+  <Pressable onPress={() => onContentPressed(item)}>
+    <Text style={styles.domainItemTitle}>{item.content.title}</Text>
+    {item.content.image ? (
+      <Image source={{uri: item.content.image}} style={styles.domainImage} />
+    ) : (
+      <Image source={NewsEmptyState} style={styles.domainImageEmptyState} />
+    )}
+    <View style={styles.domainTextContainer}>
+      <Text style={styles.domainItemDescription}>{item.content.description}</Text>
+    </View>
+    {/* <Gap height={14} /> */}
+  </Pressable>
 );
 
 const styles = StyleSheet.create({
-    domainItemTitle: {
-        fontSize: 16,
-        fontFamily: fonts.inter[700],
-        lineHeight: 24,
-        paddingHorizontal: 20,
-        marginTop: 12,
-        marginBottom: 12
-    },
-    domainItemDescription: {
-        fontFamily: fonts.inter[400],
-        fontSize: 16,
-        lineHeight: 24,
-    },
-    domainTextContainer: {
-        paddingHorizontal: 20,
-    },
-    domainIndicatorContainer: {
-        marginLeft: -4,
-    },
-    domainImage: {
-        height: 200,
-        marginBottom: 14
-    },
-    domainImageEmptyState: {
-        height: 135,
-        marginBottom: 14,
-        backgroundColor: colors.lightgrey,
-        resizeMode: 'center',
-        width: '100%'
-    },
+  domainItemTitle: {
+    fontSize: 16,
+    fontFamily: fonts.inter[700],
+    lineHeight: 24,
+    paddingHorizontal: 20,
+    marginTop: 12,
+    marginBottom: 12
+  },
+  domainItemDescription: {
+    fontFamily: fonts.inter[400],
+    fontSize: 16,
+    lineHeight: 24
+  },
+  domainTextContainer: {
+    paddingHorizontal: 20
+  },
+  domainIndicatorContainer: {
+    marginLeft: -4
+  },
+  domainImage: {
+    height: 200,
+    marginBottom: 14
+  },
+  domainImageEmptyState: {
+    height: 135,
+    marginBottom: 14,
+    backgroundColor: COLORS.lightgrey,
+    resizeMode: 'center',
+    width: '100%'
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the styles are consistent and follow the new color scheme. However, I noticed that `domainIndicatorContainer` style is not used anywhere in the component. If it's not needed, consider removing it to keep the code clean.
```diff
  domainIndicatorContainer: {
    marginLeft: -4
  },
```
If this style is used in other parts of your application, ignore this comment.


<!-- This is an auto-generated comment by OSS CodeRabbit -->